framework/components/ecl-social-icons/ecl-social-icons.twig

```diff
@@ -1,172 +1,44 @@
-    <li class='ecl-icon-social ecl-icon-social--linkedin'>Linkedin</li>
-    <li class='ecl-icon-social ecl-icon-social--live'>Live</li>
-    <li class='ecl-icon-social ecl-icon-social--meneame'>Meneame</li>
-    <li class='ecl-icon-social ecl-icon-social--misterwong'>Misterwong</li>
-    <li class='ecl-icon-social ecl-icon-social--myspace'>Myspace</li>
-    <li class='ecl-icon-social ecl-icon-social--netlog'>Netlog</li>
-    <li class='ecl-icon-social ecl-icon-social--netvibes'>Netvibes</li>
-    <li class='ecl-icon-social ecl-icon-social--newsvine'>Newsvine</li>
-    <li class='ecl-icon-social ecl-icon-social--nujij'>Nujij</li>
-    <li class='ecl-icon-social ecl-icon-social--pinterest'>Pinterest</li>
-    <li class='ecl-icon-social ecl-icon-social--pocket'>Pocket</li>
-    <li class='ecl-icon-social ecl-icon-social--reddit'>Reddit</li>
-    <li class='ecl-icon-social ecl-icon-social--rss'>Rss</li>
-    <li class='ecl-icon-social ecl-icon-social--share'>Share</li>
-    <li class='ecl-icon-social ecl-icon-social--skype'>Skype</li>
-    <li class='ecl-icon-social ecl-icon-social--slashdot'>Slashdot</li>
-    <li class='ecl-icon-social ecl-icon-social--sonico'>Sonico</li>
-    <li class='ecl-icon-social ecl-icon-social--spotify'>Spotify</li>
-    <li class='ecl-icon-social ecl-icon-social--storify'>Storify</li>
-    <li class='ecl-icon-social ecl-icon-social--studivz'>Studivz</li>
-    <li class='ecl-icon-social ecl-icon-social--stumbleupon'>Stumbleupon</li>
-    <li class='ecl-icon-social ecl-icon-social--technorati'>Technorati</li>
-    <li class='ecl-icon-social ecl-icon-social--tuenti'>Tuenti</li>
-    <li class='ecl-icon-social ecl-icon-social--tumblr'>Tumblr</li>
-    <li class='ecl-icon-social ecl-icon-social--twitter'>Twitter</li>
-    <li class='ecl-icon-social ecl-icon-social--viadeo'>Viadeo</li>
-    <li class='ecl-icon-social ecl-icon-social--vine'>Vine</li>
-    <li class='ecl-icon-social ecl-icon-social--wordpress'>Wordpress</li>
-    <li class='ecl-icon-social ecl-icon-social--wykop'>Wykop</li>
-    <li class='ecl-icon-social ecl-icon-social--xerpi'>Xerpi</li>
-    <li class='ecl-icon-social ecl-icon-social--yammer'>Yammer</li>
-    <li class='ecl-icon-social ecl-icon-social--youtube'>Youtube</li>
-    <li class='ecl-icon-social ecl-icon-social--print'>Print</li>
-  </ul>
-</div>
+  2) Variables building a single link:
+  - link: (structure) Obligatory data for the link itself.
+  {
+    target: (string) Obligatory link target.
+    modifier: (string) Obligatory modifier.
+    extraClasses: (string) Optional classes on top of defaults.
+    title: (string) Obligatory link title.
+  }
+
+  3) Variables building a span - this is a default output if the preceding fail.
+  - modifier: (string) Obligatory modifier.
+  - text: (text) The textual value.
+#}
+
+{# If there is a list, make a list #}
+{% if list is defined and list is iterable %}
+<ul class="{{ listExtraClasses ?: 'ecl-list' }}">
```

Comment:
Not sure about that. I thought the component was just about adding a social icon, not a whole list of links ^^'